### DISCUSSION TITLE
How to manual translate admin ui?

### DISCUSSION THREAD

**Discussion by c001os** [directus/directus#2807] [Archive]
Hi!

I want to manually translate admin ui to hungarian. What is the right way? When i translate in https://locales.directus.io/project/directus/hu how to apply the translated content?

Thank You

**Comment by rijkvanzanten** [comment]
Hi there!


You translate it on https://locales.directus.io/project/directus/hu, Crowdin will open a PR like this https://github.com/directus/app/pull/2816, we'll merge and release it to the world 🙂
